Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "iomrascálaí"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
We can do that, but could you please then adjust the cards on Trello? I was thinking of having 0.1.0 as our first real release when the time keeping stuff is implemented and the bot is running successfully on CGOS. But we can also make this 0.2.0 and adjust everything on Trello.